packages/react/fast-components-react-msft/package.json

```diff
@@ -106,7 +106,7 @@
     "focus-visible": "^4.1.5",
     "jest": "^25.4.0",
     "lodash-es": "^4.0.0",
-    "prettier": "2.0.2",
+    "prettier": "2.0.5",
```

Comment:
Is there a reason we're targeting a specific version and not latest?

Reply:
@nicholasrice prettier should always be fixed or we'll hit build breaks as they change/update/fix formatting issues as they see fit.

Reply:
It'll require a code change just for formatting which seems unnecessary. I'm not sure updating to latest here is necessary even as there are no changes from the version we are currently locked to.

Reply:
I think it's prudent to maintain up-to-date packages. I also disagree that we should fix any package.  It's much easier to troubleshoot issues when they happen as a result of a release, otherwise, it's like finding a needle in a haystack.

Reply:
To be super clear - we will NOT have issues if we fix this package. We 100% have issues with we Semver this package.

Reply:
Prettier calls this practice out specifically in their docs: https://prettier.io/docs/en/install.html

Reply:
By design.  We have always operated on the premise that keeping up-to-date is preferred and build breaks as a result of package dependencies is an acceptable risk.

Reply:
In that case, I'm going to hold on the PR. I fundamentally believe that the current build break is unrelated to this. This PR will introduce build breaks that are 100% avoidable by following previous precedence with prettier prior to the 2.0 update as well as prettier's own guidance.

Reply:
I am not suggesting this fixes the build break. I'm am suggesting that "100% avoidable" is not accurate, given the other bug filed on build break related to prettier.